src/qt/locale/bitcoin_fr.ts

```diff
@@ -1407,226 +1407,230 @@
     </message>
     <message>
         <source>Helpful tip of the day:</source>
-        <translation type="unfinished"></translation>
+        <translation>Conseil utile du jour :</translation>
     </message>
     <message>
         <source>Tip</source>
-        <translation type="unfinished"></translation>
+        <translation>Conseil :</translation>
     </message>
     <message>
         <source>Never share your wallet.dat file/your private key with anyone</source>
-        <translation type="unfinished"></translation>
+        <translation>Ne partagez jamais votre fichier wallet.dat/votre clé privée avec quiconque.</translation>
```

Comment:
"Ne jamais partager.." sounds better.

Reply:
"Ne partagez jamais" emphasises the negation and the imperative to show that it is an instruction.